src/data/qpf/univariate/qpf.lean

```diff
@@ -0,0 +1,796 @@
+/-
+Copyright (c) 2018 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jeremy Avigad
+
+We assume the following:
```

Comment:
-> module doc string

Please cite your ITP paper.

Reply:
Will do. What is the accepted style for citations?

Reply:
I think it is adding the reference to `docs/references.bib` and then add a link like this:
```
* [Mario Carneiro, *Formalizing computability theory via partial recursive functions*][carneiro2019]
```